### PR TITLE
adds async-await lesson

### PR DESCRIPTION
**Pull request by I3uckwheat**
Please let me know if you see any typos or if something could be written better!

### CODE REVIEW

File: async-apis/async-await.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+</script>
+~~~
+Since `response` is still the same object we have passed to the `.then()` block at the start, we still need to use the `.json()` method, which in turn returns a promise. Because `.json()` returns a promise, we can use `await` to assign the response to a variable.
+~~~javascript
+<script>
+  const img = document.querySelector('img')
+
+  async function getCats() {
+    const response = await fetch('https://api.giphy.com/v1/gifs/translate?api_key=111111&s=cats', {mode: 'cors'});
+    const catData = response.json();
```
Comment (codyloyd) [inline async-apis/async-await.md]: does this line not need an await?

**Comment by KevinMulhern** [comment]
👍 LGTM

**Comment by codyloyd** [comment]
MRG -> @KevinMulhern would you twiddle the seeds file to get this in?

I think it should go right before the project here 
image

**Comment by I3uckwheat** [comment]
@Karmaqueen84 Yeah, i can fix that quick!

**Pull request merged**